rocky/tools/view_helpers.py

```diff
@@ -153,3 +156,24 @@ def build_breadcrumbs(self):
                 "text": _("Objects"),
             }
         ]
+
+
+def schedule_task(request: HttpRequest, queue_name: str, item: QueuePrioritizedItem) -> None:
```

Comment:
It's easier on the eye to just use the shortcuts for adding messages (`messages.info`, `messages.error`, `messages.success`, ...). See also [this](https://docs.djangoproject.com/en/4.2/ref/contrib/messages/#adding-a-message)